components/engine/libcontainerd/client_local_windows.go

```diff
@@ -70,6 +71,10 @@ const (
 // of docker.
 const defaultOwner = "docker"
 
+func (c *client) Version(ctx context.Context) (containerd.Version, error) {
+	return containerd.Version{}, errors.New("not implemented on Windows")
```

Comment:
this will not be a problem on docker for windows, will it? @johnstep @gtardif

Reply:
no it will simply display `"N/A"` from `components/engine/daemon/info_unix.go L55`